app/models/manageiq/providers/oracle_cloud/cloud_manager/cloud_database.rb

```diff
@@ -164,4 +165,23 @@ def self.create_mysql_database(ext_management_system, options)
       )
     end
   end
+
+  def raw_delete_cloud_database
+    db_engine.include?("Oracle Database") ? delete_oracle_database : delete_mysql_database
```

Comment:
Seemed like the best way to differentiate between the two types of databases based on https://github.com/ManageIQ/manageiq-providers-oracle_cloud/blob/master/app/models/manageiq/providers/oracle_cloud/inventory/parser.rb#L55-L74

Reply:
Agree this is the best way to differentiate, could we pull this out into its own method like `db_type` and return `:mysql` or `:oracle` ?

Reply:
Sure, I'll make the change